collect_app/src/androidTest/java/org/odk/collect/android/AllWidgetsFormTest.java

```diff
@@ -515,32 +526,65 @@ public void testBearingWidget() {
 
     public void testRangeIntegerWidget() {
 
-         Screengrab.screenshot("range-integer");
+        int randomValue = randomInt() % 9;
+        onVisibleSeekBar().perform(setProgress(randomValue));
 
-         onView(withText("Range integer widget")).perform(swipeLeft());
+        Screengrab.screenshot("range-integer");
+
+        openWidgetList();
+        onVisibleListView().atPosition(12).perform(click());
```

Comment:
Nice to see these coming together, @ArmenApresyan8! I would tend to avoid using position here because it will be annoying to change when more questions are added to the all widgets form. Why not use the same strategy as the previously-existing tests?

Reply:
I'v tried to use the same strategy, but the test was failing with an error, because there are 2 fields in the list with name "Range integer widget" so it can't decide which one to choose.

Reply:
Ahhh! Well that makes sense. I think it would be most helpful to change the all widgets form to make the labels more specific, then (e.g. Range vertical integer widget). Can you please change it in the XML and comment on the Google sheet linked from the README so that someone with edit privileges can update the source?

Reply:
Sure!

Reply:
OK I have commented on the Google sheets, but what about the XML? Can we edit it?

Reply:
The All widgets form is stored both in Google Drive and Aggregate server?

Reply:
Awesome, thanks! For this test it’s in the source. You can change it as part of this PR.